File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/Segment.kt
package com.sdkwork.ai

data class Segment(
    val id: Int? = null,
    val seek: Int? = null,
    val start: Double? = null,
    val end: Double? = null,
    val text: String? = null,
    val tokens: List<Int>? = null,
    val temperature: Double? = null,
    val avgLogprob: Double? = null,
    val compressionRatio: Double? = null,
    val noSpeechProb: Double? = null
)

File: sdkwork-ai-sdk-csharp/Models/Segment.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class Segment
    {
        public int? Id { get; set; }
        public int? Seek { get; set; }
        public double? Start { get; set; }
        public double? End { get; set; }
        public string? Text { get; set; }
        public List<int>? Tokens { get; set; }
        public double? Temperature { get; set; }
        public double? AvgLogprob { get; set; }
        public double? CompressionRatio { get; set; }
        public double? NoSpeechProb { get; set; }
    }
}
